include/deal.II/base/vectorization.h

```diff
@@ -3841,9 +3841,25 @@ inline DEAL_II_ALWAYS_INLINE VectorizedArray<Number, width>
   return VectorizedArray<Number, width>() - u;
 }
 
+/**
+ * Output operator for vectorized array.
+ *
+ * @relatesalso VectorizedArray
+ */
+template <typename Number>
+inline std::ostream &
+operator<<(std::ostream &out, const dealii::VectorizedArray<Number> &p)
+{
```

Comment:
Please generalize it, such that it works for arbitrary `VectorizedArray<Number, width>`:
```cpp
template <typename Number, int width>
inline std::ostream &
operator<<(std::ostream &out, const dealii::VectorizedArray<Number, width> &p)
```